var/spack/repos/builtin/packages/py-cdat-lite/package.py

```diff
@@ -40,3 +40,14 @@ class PyCdatLite(PythonPackage):
     depends_on("python@2.5:2.8", type=('build', 'run'))
     depends_on("py-numpy", type=('build', 'run'))
     depends_on('py-setuptools', type='build')
+
+    phases = ['install']
+
+    def install(self, spec, prefix):
+        """Install everything from build directory."""
+        install_args = self.install_args(spec, prefix)
+        # Combine all phases into a single setup.py command,
+        # otherwise extensions are rebuilt without rpath by install phase:
```

Comment:
Wait, really? I wouldn't expect that behavior. Maybe I'll have to test this one.

Btw, could this routine theoretically be used to install any Python package? Like in `PythonPackage` can we always run this sequence of commands, and is `--rpath` always available? If that is the case, we could solve the problem you've been seeing by always adding link dependencies to the `--rpath`.